### DISCUSSION TITLE
CopyQ

### DISCUSSION THREAD

**Discussion by lawakin** [catppuccin/catppuccin#2609] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

CopyQ is not that hard to theme, it has a simple .ini file that we can import into the program.
the variables and CSS in the default theme is a bit unclear and i don't think it's possible to change the style of the scrollbar but since the built-in themes also gets rid of the scrollbar it should be fine to remove it.

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by scarcekoi** [comment]
I just made a PR: https://github.com/hluk/CopyQ/pull/3380.

**Reply by lawakin** [reply]
oh my god thank you so much
i did already wrote one but i was too embarrassed to show it to anyone (it is ugly and only mocha)

**Reply by scarcekoi** [reply]
I closed the PR in favour of hosting it on Catppuccin: https://github.com/catppuccin/catppuccin/issues/2963.

**Comment by sgoudham** [comment]
Closing as any further review should take place in #2963.
